src/app/model/Mission.ts

```diff
@@ -63,6 +64,31 @@ class Mission extends BaseModel {
 
   filterByStatus = (missions: MissionInterface[], status: MissionStatus) =>
     missions.filter((mission) => mission.status === status);
+
+  /**
+   * Given a mission object creates a new entry in firestore
+   * returns the mission id
+   * @param {object} mission
+   * @return {string}
+   */
+  async create(mission: MissionInterface): Promise<string> {
+    const missionId = uuidV4();
+    const collection = this.getCollection(this.collectionName);
+
+    //Add mission id and sanitize data
+    mission.id = missionId;
+    const sanitizedMission = this.sanitize(mission);
```

Comment:
Decided this in a separate convo, but we're gonna use `this.load`(maybe needs a new name?) instead for creates and save `sanitize` for updates. `load` will default any missing data on the object as well as sanitize it

Reply:
I've changed this in the new PR #259, could you have a look over there? Honestly still not 100% sure how to correctly use the `load` and `sanitize` functions.

Reply:
@mTRx0  yep! That's how I understand it. I am learning about TS on the go.

Reply:
Sorry if that was confusing. Yeh use `load` for creating new objects and fetching objects from the db and `sanitize` for updates.

Reply:
Alright, got it. Thanks!